src/styles/blocks/card.scss

```diff
@@ -0,0 +1,71 @@
+.card {
+  border: 1px solid $color-gray;
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  &__description {
+    color: $text-main-color;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+  &__code {
+    color: $text-secondary-color;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__reviews {
+    color: $text-main-color;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__stars {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 24px;
+    margin-top: 16px;
+  }
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__cost {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+  &__price {
+    color: $text-secondary-color;
+    font-size: 12px;
+    line-height: 18px;
+  }
+  &__total {
+    color: $text-main-color;
+    line-height: 18px;
+    font-weight: 700;
+  }
+
+  &__button {
+    height: 40px;
+    width: 100%;
+    border-radius: 5px;
+    font-size: 14px;
+    background-color: #00acdc;
+    color: #fff;
+    box-sizing: border-box;
+    border: 1px solid #00acdc;
+
+    &:hover {
+      color: $color-blue-accent;
+      background-color: $color-white;
```

Comment:
The background-color value for the hover state is using a variable ($color-white), which is good for consistency. However, ensure that this variable is defined in your variables file. If it's not, this could lead to unexpected results.